Applications/GesturePod/onComputer/README.md

```diff
@@ -0,0 +1,46 @@
+GesturePod *(Gesture Recognition Pipeline)*
```

Comment:
@ShishirPatil could you please check license info here

Reply:
We are not dependent on any external libraries for this directory. No code snippets used either. We have ownership on every line of code, for this directory.